### PR TITLE
Add new file

### PR DESCRIPTION
**Pull request by REYESJOHNRUIZ**

### CODE REVIEW

File: Reyes_JohnRuiz/index.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+          <li>To learn some new Skills</li>
+          <li>to travel outside of country </li>
+          <li>to </li>
+          <ol>To be happy</ol>
+        </ol>
+      </section>
+
+      <section>
+        <h2>Add a comment here</h2>
+        <form id="comment-form" class="add-comment">
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: IDs should be written in snake case. All lower case separated by underscore.
See the reference for clarifications.
```suggestion
        <form id="comment_form" class="add-comment">
```

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_JohnRuiz/index.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+          />
+          <textarea
+            name="textarea-for-comment"
+            id="comment-text"
+            placeholder="Add your comment here"
+          >
+          </textarea>
+          <button type="button" name="comment" id="submit-comment">Comment</button>
+        </form>
+      </section>
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: IDs should be written in snake case. All lower case separated by underscore. Remove excess line spacing as much as possible. See the reference and suggested code for clarifications.
```suggestion
            name="name2"
            id="comment_name"
            placeholder="Enter your name"/>
          <textarea
            name="textarea-for-comment"
            id="comment_text"
            placeholder="Add your comment here">
          </textarea>
          <button type="button" name="comment" id="submit_comment">Comment</button>
        </form>
      </section>
```

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_JohnRuiz/index.html [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+      <section id="comments-container" class="comments-of-team">
+        <h2>Sorted Buttons</h2>
+        <button type="button" id="sort-asc">Sort by Date Ascending</button>
+        <button type="button" id="sort-desc">Sort by Date Descending</button>
+        <h2>New Comments</h2>
+      </section>
+
+      <section id="comments-container" class="comments-of-team">
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: IDs should be written in snake case. All lower case separated by underscore. See the reference and suggested code for clarifications.
```suggestion
      <section id="comments_container" class="comments-of-team">
```
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_JohnRuiz/index.html [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+          to remain disciplined and dedicated to your studies to maximize the
+          benefits of this convenient learning option, especially when the cost
+          of transportation and other expenses can add up quickly.
+        </p>
+      </div>
+    </main>
+
+    <div class="Contact">
+      <h2>Contact Me</h2>
+      <form id="contact-form" class="form">
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: IDs should be written in snake case. All lower case separated by underscore. See the reference and suggested code for clarifications.
```suggestion
      <form id="contact_form" class="form">
```
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_JohnRuiz/index.html [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+          </textarea>
+          <button type="button" name="comment" id="submit-comment">Comment</button>
+        </form>
+      </section>
+
+      <section id="comments-container" class="comments-of-team">
+        <h2>Sorted Buttons</h2>
+        <button type="button" id="sort-asc">Sort by Date Ascending</button>
+        <button type="button" id="sort-desc">Sort by Date Descending</button>
+        <h2>New Comments</h2>
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: IDs should be written in snake case. All lower case separated by underscore. See the reference and suggested code for clarifications.
```suggestion
      <section id="comments_container" class="comments-of-team">
        <h2>Sorted Buttons</h2>
        <button type="button" id="sort_asc">Sort by Date Ascending</button>
        <button type="button" id="sort_desc">Sort by Date Descending</button>
        <h2>New Comments</h2>
```
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_JohnRuiz/index.html [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+          scorching heat. However, without the structure of a traditional
+          classroom, it's easy to get sidetracked and lose focus. It's crucial
+          to remain disciplined and dedicated to your studies to maximize the
+          benefits of this convenient learning option, especially when the cost
+          of transportation and other expenses can add up quickly.
+        </p>
+      </div>
+    </main>
+
+    <div class="Contact">
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: All class name should be in lower case. 
```suggestion
    <div class="contact">
```
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_JohnRuiz/script.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+    const textValue = commentTextInput.value.trim();
+    submitCommentButton.disabled = !(nameValue && textValue);
+  }
+
+  checkCommentValidity();
+
+  commentNameInput.addEventListener("input", checkCommentValidity);
+  commentTextInput.addEventListener("input", checkCommentValidity);
+
+  submitCommentButton.addEventListener("click", (e) => {
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/script.js]: Do not use arrow function as a method of an object. You can use the sample code and adjust it to your intended use
```suggestion
  submitCommentButton.addEventListener("click", function() => {
```

File: Reyes_JohnRuiz/script.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+  function addComment(name, text, date) {
+    const commentElement = document.createElement("p");
+    commentElement.innerHTML = `Name: ${name}<br>Comment: 
+        ${text}<br><small>${date.toLocaleString()}</small>`;
+    commentsContainer.appendChild(commentElement);
+
+    comments.push({ name, text, date });
+  }
+
+  sortAscButton.addEventListener("click", () => {
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/script.js]: Do not use arrow function as a method of an object. You can use the sample code and adjust it to your intended use.
```suggestion
  sortAscButton.addEventListener("click", function() => {
```

File: Reyes_JohnRuiz/script.js [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+    sortComments(true);
+  });
+
+  sortDescButton.addEventListener("click", () => {
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/script.js]: Do not use arrow function as a method of an object. You can use the sample code and adjust it to your intended use.
```suggestion
  sortDescButton.addEventListener("click", function() => {
```

**Review by Tapispisan-YvesJohn — CHANGES_REQUESTED** [review]
Please check your js script and css to reflect the changes in your html class names and IDs.

File: Reyes_JohnRuiz/index.html [hunk 10]
```diff
@@ -91,39 +91,44 @@ <h2>My Goals</h2>
...
-        <form id="comment-form" class="add-comment">
+        <form id="comment_form" class="add-comment">
           <input
             type="text"
             name="name2"
-            id="comment-name"
+            id="comment_name"
             placeholder="Enter your name"
           />
           <textarea
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: Don't separate the closing tag in a new line. 
```suggestion
            placeholder="Enter your name"/>
          <textarea
```

File: Reyes_JohnRuiz/index.html [hunk 11]
```diff
@@ -91,39 +91,44 @@ <h2>My Goals</h2>
...
             name="textarea-for-comment"
-            id="comment-text"
+            id="comment_text"
             placeholder="Add your comment here"
           >
           </textarea>
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: Don't separate the closing tag in a new line.
```suggestion
            placeholder="Add your comment here">
          </textarea>
```

File: Reyes_JohnRuiz/index.html [hunk 12]
```diff
@@ -159,26 +163,27 @@ <h2>Thoughts about online class</h2>
...
         <input type="checkbox" name="copy" id="copy" />
         <textarea
           id="message"
           name="message"
           rows="3"
           cols="15"
-          placeholder="Message">
+          placeholder="Message"
+        >
         </textarea>
```
Comment (Tapispisan-YvesJohn) [inline Reyes_JohnRuiz/index.html]: Don't separate the closing tag in a new line.
```suggestion
          placeholder="Message">
        </textarea>
```

**Review by Tapispisan-YvesJohn — CHANGES_REQUESTED** [review]
Please check your css file to ensure that the changes of your class names and IDs is reflected properly.

**Review by Tapispisan-YvesJohn — APPROVED** [review]
Great job, you can merge it now.

**Comment by REYESJOHNRUIZ** [comment]
thank you po

**Pull request merged**